src/libraries/System.Net.Http/tests/FunctionalTests/MetricsTest.cs

```diff
@@ -363,7 +415,6 @@ public Task RequestDuration_CustomTags_Recorded()
         [ConditionalTheory(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
         [InlineData("System.Net.Http.HttpRequestOut.Start")]
         [InlineData("System.Net.Http.Request")]
-        [InlineData("System.Net.Http.HttpRequestOut.Stop")]
```

Comment:
While working on .NET 8, there was a request from internal partners that metrics enrichment callbacks should be registered from DiagnosticListener handlers for `System.Net.Http.HttpRequestOut.Stop`, however in the end Metrics enrichment support did not make it into their release.

Metrics enrichment seems to be vastly underused in general, and I don't think this particular case is something anyone would ever need in practice.